src/agent/state/inspector-state.ts

```diff
@@ -325,9 +333,18 @@ class StateResolver {
   }
 
   static stripFileProtocol_(path: string) {
-    return path.toLowerCase().startsWith(FILE_PROTOCOL)
-      ? path.substr(FILE_PROTOCOL.length)
-      : path;
+    const lowerPath = path.toLowerCase();
+    // match paths like file:///C:/... on windows
+    // but do not match paths like file:///home/... on linux
+    if (RegExp(`^${WINDOWS_FILE_PROTOCOL}[a-zA-Z]+:`).test(lowerPath)) {
```

Comment:
This `RegExp` object never changes but this code path is taken often, can you make this a `static const` as well?

Reply:
Done